entries/d/dmoj.ca.json

```
@@ -0,0 +1,12 @@
+{
+  "DMOJ": {
+    "domain": "dmoj.ca",
+    "tfa": [
+      "totp"
+    ],
```

This page is linked on https://dmoj.ca/about/:
```suggestion
    ],
    "documentation": "https://dmoj.ca/post/144-2-factor-authentication",
```